renderer/views/player.js

```diff
@@ -269,9 +269,9 @@ function renderCastScreen (state) {
     castIcon = 'tv'
     castType = 'DLNA'
     isCast = true
-  } else if (state.playing.location === 'vlc') {
+  } else if (state.playing.location === 'external') {
     castIcon = 'tv'
-    castType = 'VLC'
+    castType = 'External' // TODO: Should we use the player name instead? see `unsupported-media-modal.js`
```

Comment:
I would like to have a more convenient way to get the player name, compared to how it is done in `unsupported-media-modal.js`. So I guess I will stick with 'External' for now.